service/templates/projects.html

```diff
@@ -340,6 +340,14 @@ <h2> Settings for [[displayName]]</h2>
                <input type="checkbox" name="set_private" ng-model="proj_var.is_private" ng-disabled="!validKeyValue(proj_var.name, proj_var.value)" /> <strong>Private Variable</strong>
              </label>
             </div>
+            <div>
+              <label class="control-label">
+                <input type="checkbox" name="set_restricted" ng-model="proj_var.is_restricted" ng-disabled="!validKeyValue(proj_var.name, proj_var.value)" /> <strong>Restricted Variable</strong>
+              </label>
+            </div>
+            <span class="warning-text" ng-show="proj_var.is_restricted"><i class="fa fa-warning"></i>
+              Restricted variables are only used with commands that won't leak the variables. Should also be set to private.
```

Comment:
I'm not sure they should also be set to private. They are only accessible to project admins, so it's conceivable that project admins are fine being able to read them from the UI.

Reply:
Okay I can change this warning to clarify they won't be private. I do want to make sure it's understood that Restricted is not a subset of private.

Reply:
Or I can clarify what "restricted" means (and will add to wiki), users will see I suppose that it's not redacted when they press the plus button.